solr/vufind/authority/conf/solrconfig.xml

```diff
@@ -32,7 +32,7 @@
        that you fully re-index after changing this setting as it can
        affect both how text is indexed and queried.
     -->
-  <luceneMatchVersion>9.7</luceneMatchVersion>
+  <luceneMatchVersion>9.9.2</luceneMatchVersion>
```

Comment:
I assume it doesn't make a difference, but the examples in Solr use `9.9`.

Reply:
I picked this version based on the changelog rather than the examples, but I think using the more general version from the example is probably preferable, so I have changed it. Thanks for noticing!